runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -320,32 +311,33 @@ ir::OperandIndex BaseLoader<LoaderDomain, SpecificLoader>::loadOperand(const Ten
   // Create operand
   const auto operand_index = subg.addOperand(shape, type_info);
 
-  std::cout << "Tensor : " << tensor->buffer() << std::endl;
   // Constant tensors are indicated by non-empty data.
   const auto *data = _model->buffers()->Get(tensor->buffer())->data();
 
   if (data != nullptr)
   {
     auto ptr = std::make_unique<ir::CachedData>(data->data(), data->size());
 
-    int32_t offset_start = const_cast<unsigned char *>(data->data()) - _base;
-    int32_t offset_end = offset_start + data->size();
-
-    int32_t aligned_start =
-        offset_start % _pagesize == 0 ? offset_start : ((offset_start / _pagesize) + 1) * _pagesize;
-    int32_t aligned_end =
-        offset_end % _pagesize == 0 ? offset_end : ((offset_end / _pagesize) - 1) * _pagesize;
-
-    std::cout << "Offset start(" << offset_start << "," << aligned_start << ")" << std::endl;
-    std::cout << "Offset end(" << offset_end << "," << aligned_end << ")" << std::endl;
-
-    if (aligned_end > aligned_start)
+    // Calculate offset from base address of mapped region
+    int32_t unaligned_offset_start = const_cast<unsigned char *>(data->data()) - _base;
+    int32_t unaligned_offset_end = unaligned_offset_start + data->size();
+
+    // Calculated aligned offset from base address of mapped region
+    // munmap accepts memory address which is a multiple of the pagesize
+    int32_t aligned_offset_start = unaligned_offset_start % _pagesize == 0
+                                       ? unaligned_offset_start
+                                       : ((unaligned_offset_start / _pagesize) + 1) * _pagesize;
+    int32_t aligned_offset_end = unaligned_offset_end % _pagesize == 0
+                                     ? unaligned_offset_end
+                                     : ((unaligned_offset_end / _pagesize) - 1) * _pagesize;
```

Comment:
What about this one-liner?

```suggestion
    int32_t aligned_offset_end = (unaligned_offset_end / _pagesize) * _pagesize;
```

Reply:
By the way, `size` should be multiple of the page size as well?

Reply:
@wateret It doens't have to be multiple of the page size. However, for correct execution, it should be multiple of the page size.
`munmap` frees all pages containing a part of the indidated range. If we use unaligned size for `munmap`, the last page may have weights that haven't been copied yet. When we access weight in unmapped area, program will generate `SIGSEGV` error.
To avoid this error, `size` should be multiple of the page size.